docs/dcp2_system_design.rst

```diff
@@ -986,6 +1065,400 @@ Broad Institute).
 
 
 
+DCP data releases
+=================
+
+A data release of the DCP consists of a set of disjunctive TDR snapshots that
+provides a coherent and immutable view of the contents of the DCP at the time
+the data release is published.
+
+Some of those snapshots may contain meta(data) for exactly one HCA project
+while other snapshots may contain (meta)data for more than one HCA project.
+
+**Completeness**: Each snapshot in the release must be complete; that is,
+every entity referenced by every subgraph in the snapshot must also be present
+in that same snapshot. Expressed in terms of the TDR schema this means that
+for every row in the ``links`` table of the snapshot, and any reference *u* to
+an entity of type *T* by the JSON structure in the ``contents`` column of that
+row, there must be exactly one row index *i* in table *T* of that snapshot for
+which *T[i].T_id = u*. Similarly, for every row index *j* in the ``links``
+table of the snapshot, there must be exactly one row index *i* in the
+``project`` table of that snapshot for which *project[i].project_id =
+links[j].project_id*.
+
+**Disjunctivity**: No metadata entity may be included in more than one of the
+snapshots that make up the data release. Expressed in terms of the TDR schema
```

Comment:
What about biomaterials? The same donor may be used across multiple projects. I think we have examples from HDBR. Or is this just at the project level?